upf/grpc/upf_pb2_grpc.py

```diff
@@ -0,0 +1,66 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
Does it make sense to put auto-generated stuff in a dedicated package? Then we can import them with `__init__.py` to make this transparent

Reply:
Sounds reasonable. One issue I have with the protobuf generation right now is that it actually has to be modified for the imports to work.

Reply:
Moved to sub-module called `generated`.